includes/html/graphs/application/ntp-server_bits.inc.php

```diff
@@ -3,7 +3,7 @@
 $nototal = 1;
 $ds_in = 'packets_recv';
 $ds_out = 'packets_sent';
-$graph_title .= '::packets';
+$graph_title = '::packets';
```

Comment:
This doesn't seem correct.  It should be appending to an existing title.  a title starting with :: makes no sense.

Reply:
then there is missing a file include on head of file, so $graph_title string addition is working

Reply:
sure, then set $graph_title like this and leave this to append:
`$graph_title ??= '';`

Reply:
```suggestion
$graph_title ??= 'NTP'; 
$graph_title .= '::packets';
```